src/SumoCoders/FrameworkCoreBundle/Resources/assets/icon-embed/logo-application.svg

```diff
@@ -1,227 +1,227 @@
-	<path fill="#EAC664" d="M128.103,36.939l0.081-0.118c-31.544,0-57.112,25.575-57.112,57.115c0,17.249,7.653,32.714,19.743,43.18
-		c0.002,0.011,0.009,0.013,0.013,0.02c1.231,1.14,2.071,2.069,1.809,3.431v0.011c-0.235,0.931-0.361,1.911-0.361,2.912
-		c0,6.599,5.347,11.95,11.948,11.95c6.597,0,11.947-5.351,11.947-11.95c0-3.106-1.193-5.939-3.139-8.062
-		c-0.005-0.002-0.005-0.002-0.008-0.007c-0.742-0.887-1.324-1.618-1.324-2.758V77.831V60.598l6.445-9.296h18.878l6.514,9.403v17.125
-		v54.93c0,1.145-0.584,1.872-1.328,2.763c0,0-0.002,0.002-0.002,0.009c-1.95,2.124-3.139,4.949-3.139,8.062
-		c0,6.597,5.349,11.949,11.945,11.949c6.602,0,11.951-5.353,11.951-11.949c0-1.01-0.13-1.985-0.363-2.912
-		c0-0.002-0.002-0.002-0.002-0.013c-0.258-1.358,0.575-2.292,1.808-3.428c0.005-0.01,0.009-0.015,0.016-0.021
-		c12.093-10.47,19.743-25.931,19.743-43.188C184.165,62.854,159.161,37.501,128.103,36.939"/>
-	<path fill="#896F00" d="M135.328,13.604c0,4.258-3.453,7.709-7.709,7.709c-4.259,0-7.708-3.451-7.708-7.709
-		c0-4.253,3.448-7.704,7.708-7.704C131.875,5.899,135.328,9.35,135.328,13.604"/>
-	<path fill="#EFD081" d="M157.899,46.078c0,16.728-13.555,30.282-30.282,30.282c-16.722,0-30.281-13.555-30.281-30.282
-		c0-16.718,13.559-30.272,30.281-30.272C144.344,15.806,157.899,29.36,157.899,46.078"/>
-	<path fill="#FFFFFF" d="M119.291,56.021c0.989,3.674,4.339,6.384,8.33,6.384c3.989,0,7.337-2.71,8.324-6.384H119.291z"/>
-	<path fill="#222222" d="M109.712,51.576c0-1.392,1.13-2.524,2.526-2.524c1.392,0,2.522,1.132,2.522,2.524H109.712z"/>
-	<path fill="#222222" d="M140.473,51.576c0-1.392,1.132-2.524,2.524-2.524c1.394,0,2.526,1.132,2.526,2.524H140.473z"/>
-	<path fill="#FFFFFF" d="M58.951,49.004c0,1.877-1.526,3.403-3.403,3.403c-1.882,0-3.406-1.526-3.406-3.403
-		c0-1.882,1.524-3.406,3.406-3.406C57.425,45.598,58.951,47.122,58.951,49.004"/>
-	<path fill="#222222" d="M68.605,41.575v-0.009c-0.011-0.833-0.687-1.503-1.524-1.503c-0.834,0-1.51,0.671-1.521,1.503v0.002
-		c0,0.006-0.002,0.011-0.002,0.018h0.002c0.013,0.83,0.691,1.502,1.521,1.502c0.837,0,1.513-0.672,1.524-1.502V41.575z"/>
-	<path fill="#222222" d="M45.535,41.579c0-0.004,0.002-0.011,0.002-0.013h-0.002c-0.011-0.833-0.689-1.504-1.524-1.504
-		c-0.837,0-1.51,0.672-1.519,1.504h-0.004c0,0.002,0.002,0.007,0.002,0.009c0,0.002-0.002,0.004-0.002,0.011h0.004
-		c0.008,0.83,0.682,1.501,1.519,1.501c0.835,0,1.516-0.671,1.524-1.501V41.579z"/>
-	<polygon fill="#222222" points="201.83,59.115 199.676,64.229 197.52,59.115 	"/>
-</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Orange" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="255.247px" height="162.711px" viewBox="0 0 255.247 162.711" enable-background="new 0 0 255.247 162.711"
```

Comment:
Ik meen me te herinneren dat svg's soms beter gerendered worden in browsers als hun groottes ronde getallen zijn en dus geen komma's bevatten. (ook al is dit niet echt relevant in deze PR)